### PR TITLE
Add Github link

### LINKED ISSUE #278: Add a link / icon / some pointer to github on the homepage
Probably not the generic "fork me on github" banner, maybe just a link in the title navigation bar or something, but it'd be nice to have a link

### PR DESCRIPTION
**Pull request by msavich**
# Advanced HW 5 submission

Closes #278 

Name: Mel Savich
Uniqname: msavich

### CODE REVIEW

**Comment by ppannuto** [comment]
Have you tried this on multiple screen sizes (i.e. mobile, tablet)? iirc getting the header to re-size nicely was a bit sensitive to what's in it, and I don't think you can just add text without some css love as well.

**Comment by msavich** [comment]
Is there a simple way to check how the website looks on a phone/tablet/whatever when I'm running it on a server? I tried some websites that preview a url on a phone/tablet and plugged in the server address as the url, and the header looked fine on everything. I also connected my laptop to another screen and it looked ok.

**Comment by ppannuto** [comment]
https://developers.google.com/web/tools/chrome-devtools/device-mode/emulate-mobile-viewports

**Comment by ppannuto** [comment]
Also just resizing your browser window is probably fine - there are 4 sizes
/ layouts that it should jump between [it uses bootstrap]

File: _includes/header.html [hunk 1]
```diff
@@ -48,6 +48,7 @@
           <li class="hidden-sm"><a href="/#grading">Attendance / Grading / Homework</a></li>
           <li class="hidden-xs hidden-md hidden-lg"><a href="/#grading">Policies</a></li>
           <li><a href="/reference">Reference</a></li>
+          <li><a href="https://github.com/c4cs/c4cs.github.io">Github</a></li>
```
Comment (mterwill) [inline _includes/header.html]: GitHub should have a capital H.

**Comment by msavich** [comment]
Could I have the navbar collapse sooner than usual? Before it hits two lines? Or would you like the text to scale, and have it collapse at its usual size

**Comment by ppannuto** [comment]
Whatever you think looks better.

One cute trick I've seen is to do something like:

<github logo> GitHub

in the text bar and then shrink to just the logo (no text) when stuff won't fit. Should be easy to do by styling the image and text with different CSS

**Comment by msavich** [comment]
Still working on the sizing issue

**Comment by msavich** [comment]
Let me know what you guys think! or if you want me to edit the style of the code some other way

**Comment by msavich** [comment]
thinking about using font awesome for the GitHub icon rather than a static image

**Comment by mterwill** [comment]
LGTM! While I'm a huge fan of FontAwesome, I don't think it's worth including in the course website at this point. Thanks for all of your work on this. Credit granted.

**Pull request merged**